packages/runtime/src/reflection-info.ts

```diff
@@ -37,6 +37,9 @@ import type {JsonValue} from "./json-typings";
  * Because the JSON format requires the original value name, we store the
  * dropped prefix here, so that the JSON format implementation can restore
  * the original value names.
+ * 
+ * 
+ * [3] the options for the enum and enum values (optional)
  */
 export type EnumInfo = readonly [
```

Comment:
This is a rather unfortunate structure compared to the `MessageInfo`. Maybe in v3 this change be updated to an object instead of an array/tuple.